interview/infra/ocr/tesseract_client.py

```
@@ -0,0 +1,34 @@
+import pytesseract
+from pdf2image import convert_from_path
+from PIL import Image
+import os
+import tempfile
+
+
+class TesseractOCRClient:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)